endpoints-frameworks-v2/backend/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!--
+ Copyright 2016 Google Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<web-app xmlns="http://java.sun.com/xml/ns/javaee"  version="2.5">
+    <servlet>
```

Comment:
Some comments about this would be good -- it's one of the points of your sample, bring some attention here.